modular_skyrat/modules/primitive_catgirls/code/clothing.dm

```diff
@@ -136,3 +136,8 @@
 
 /obj/item/forging/reagent_weapon/axe/fake_copper
 	custom_materials = list(/datum/material/copporcitite = SHEET_MATERIAL_AMOUNT)
+
+/obj/item/climbing_hook/strong
+	name = "strong climbing hook"
+	desc = "A climbing hook built to high quality, you aren't likely to break this thing in your lifetime."
+	uses = INFINITY
```

Comment:
What's this for?

Wasn't mentioned in the PR at all, I don't like stuff being added without being documented.

Reply:
not entirely sure TBH im just removing it